.github/workflows/velox_be.yml

```diff
@@ -44,33 +44,17 @@ concurrency:
 
 jobs:
   velox-ubuntu2004-test-spark32:
-    runs-on: velox-self-hosted
+    runs-on: velox-self-hosted-image
     steps:
       - uses: actions/checkout@v2
       - name: Setup docker container
         run: |
-          EXTRA_DOCKER_OPTIONS="--name velox-ubuntu2004-test-$GITHUB_RUN_ID -e NUM_THREADS=30 --detach" \
-          NON_INTERACTIVE=ON \
-          MOUNT_MAVEN_CACHE=OFF \
-          OS_IMAGE=ubuntu:20.04 \
-          tools/gluten-te/ubuntu/cbash.sh sleep 14400
-      - name: Setup maven cache
-        run: |
-          docker cp ~/.m2/repository velox-ubuntu2004-test-$GITHUB_RUN_ID:/root/.m2/
-      - name: Build Gluten velox third party
-        run: |
-          docker exec velox-ubuntu2004-test-$GITHUB_RUN_ID bash -c '
-          cd /opt/gluten/ep/build-arrow/src && \
-          ./get_arrow.sh  && \
-          ./build_arrow.sh && \
-          cd /opt/gluten/ep/build-velox/src && \
-          ./get_velox.sh && \
-          ./build_velox.sh --build_benchmarks=ON'
+          docker run --rm --init --privileged --ulimit nofile=65536:65536 --ulimit core=-1 --security-opt seccomp=unconfined -v $PWD:/opt/gluten --name velox-ubuntu2004-test-$GITHUB_RUN_ID -e NUM_THREADS=30 -dit 10.0.2.4:5000/gluten-dev/ubuntu:20.04
       - name: Build Gluten CPP library
         run: |
           docker exec velox-ubuntu2004-test-$GITHUB_RUN_ID bash -c '
           cd /opt/gluten/cpp && \
-          mkdir build && cd build && cmake -DBUILD_TESTS=ON -DBUILD_EXAMPLES=ON -DBUILD_BENCHMARKS=ON .. && make -j'
+          ./compile.sh --build_velox_backend=ON --arrow_home=/opt/arrow --velox_home=/opt/velox --build_tests=ON --build_benchmarks=ON --enable_s3=ON  --enable_hdfs=ON'
```

Comment:
Question: Why do we want to switch to ```compile.script```? Previously, Velox build and Gluten CPP build are separated jobs.